packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -474,6 +458,41 @@ export abstract class ScmElement<P extends ScmElement.Props = ScmElement.Props>
         }
     };
 
+    protected getSelectionArgs(selectedNodes: Readonly<SelectableTreeNode[]>): any[] {
```

Comment:
I wonder whether it should rather belong to the model, i.e. `widget.model.selectedResources`.

Also `ScmResource[]` instead of `any[]`. Not sure why eslint does not complain here.

Reply:
I've change 'any' to ScmResource.

It would certainly make sense to move this to the model.  However to avoid duplicated code I would have had to move collectResources to perhaps the ScmTreeModel namespace, then deal with findGroup, by which time it wasn't clear this was any better, so I've left this as is.